internal/explorer/helpers.go

```diff
@@ -270,18 +270,18 @@ func (a *App) handleStatsRequestsQueryParams(r *http.Request) (types.StatsFilter
 
 // getNodeData is a helper function that wraps fetch node data
 // it caches the results in redis to save time
-func (a *App) getNodeData(nodeIDStr string) (types.NodeWithNestedCapacity, error) {
+func (a *App) getNodeData(nodeIDStr string) (types.Node, error) {
 	nodeID, err := strconv.Atoi(nodeIDStr)
 	if err != nil {
-		return types.NodeWithNestedCapacity{}, errors.Wrap(ErrBadGateway, fmt.Sprintf("invalid node id %d: %s", nodeID, err.Error()))
+		return types.Node{}, errors.Wrap(ErrBadGateway, fmt.Sprintf("invalid node id %d: %s", nodeID, err.Error()))
```

Comment:
please make sure to encode nodeIDStr instead of nodeID in the error, also is there a reason for ErrBadGateway instead of fmt.Errorf?